errors/no-html-link-for-pages.md

```diff
@@ -1,8 +1,10 @@
 # No HTML link for pages
 
+> Prevent usage of `<a>` elements to navigate to Next.js pages.
```

Comment:
```suggestion
> Prevent usage of `<a>` elements to navigate to internal Next.js pages.
```

Maybe makes it more clear external links are okay 👌

Reply:
Will fix in rule and rule table too.